examples/performance.jl

```diff
@@ -143,7 +143,8 @@ for block_dims in ((TILE_DIM, TILE_DIM), (TILE_DIM*TILE_DIM, 1), (1, TILE_DIM*TI
             # compile kernel
             kernel(input, output, ndrange=size(output))
             for rep in 1:nreps
-            KernelAbstractions.synchronize(backend)
+                KernelAbstractions.synchronize(backend)
```

Comment:
There should be a kernel launch before the sync

Reply:
Fixing it.